misc/acrn-config/schema/VMtypes.xsd

```diff
@@ -80,21 +94,27 @@
   </xs:sequence>
 </xs:complexType>
 
-<xs:simpleType name="VmKernelType">
+<xs:simpleType name="VmKernelType" >
```

Comment:
I don't think we need any additional space here.

Reply:
removed